jina/executors/indexers/keyvalue.py

```diff
@@ -40,15 +40,21 @@ def __init__(self, path, key_length):
         def close(self):
             self._body.close()
 
-    def get_add_handler(self):
+    def get_add_handler(self) -> 'WriteHandler':
+        """Get write file handler.
+        """
         # keep _start position as in pickle serialization
         return self.WriteHandler(self.index_abspath, 'ab')
 
-    def get_create_handler(self):
+    def get_create_handler(self) -> 'WriteHandler':
+        """Get write file handler.
+        """
         self._start = 0  # override _start position
         return self.WriteHandler(self.index_abspath, 'wb')
 
-    def get_query_handler(self):
+    def get_query_handler(self) -> 'ReadHandler':
+        """Get read file handler.
+        """
```

Comment:
One-line Docstrings should be in one line. https://docs.jina.ai/chapters/docstring/docstring.html

Reply:
@Yongxuanzhang nice catch! The pr is merged already, but I will have that in mind next time.